plugins/slack/mask.yaml

```
@@ -15,136 +15,267 @@ collection:
       users:
-        alias: "Users"
+        alias: Users
         required: true
         description: "Comma separated lists of users. If only one user is included, this creates a 1:1 DM.  The ordering of the users is preserved whenever a multi-person direct message is returned. Supply a `channel` when not supplying `users`."
         index: 3
+
   files_upload:
     alias: SendFile
     parameters:
       title:
-        alias: "Title"
+        alias: Title
         description: "Title of file."
         index: 1
       initial_comment:
-        alias: "Message"
+        alias: Message
         description: "The message text introducing the file in specified `channels`."
         index: 2
       channels:
-        alias: "Channels"
+        alias: Channels
         required: true
         description: "Comma-separated list of channel names or IDs where the file will be shared."
         index: 3
       content:
-        alias: "Content"
+        alias: Content
         required: true
         description: "File contents via a POST variable."
         type: textarea
         index: 4
       filetype:
-        alias: "File Type"
+        alias: File Type
         index: 5
       filename:
-        alias: "File Name"
+        alias: File Name
         description: "Filename of file."
         index: 6
+
   reactions_add:
     alias: AddReaction
     parameters:
       channel:
-        alias: "Channel ID"
+        alias: Channel ID
         required: true
         description: "Channel where the message to add reaction to was posted."
         index: 1
       name:
-        alias: "Name"
+        alias: Name
         required: true
         description: "Reaction (emoji) name."
         index: 2
       timestamp:
-        alias: "Timestamp"
+        alias: Timestamp
         required: true
         description: "Timestamp of the message to add reaction to."
+        type: date_epoch
         index: 3
+
   users_lookupByEmail:
     alias: LookupUserByEmail
     parameters:
       email:
-        alias: "Email"
+        alias: Email
         required: true
-        description: "An email address belonging to a user in the workspace"
-        index: 1
\ No newline at end of file
+        description: "An email address belonging to a user in the workspace."
+        index: 1
+
+  users_conversations:
+    alias: ListUserChannels
+    parameters:
+      user:
+        alias: User ID
+        required: true
+        description: "Browse conversations by a specific user ID's membership. Non-public channels are restricted to those where the calling user shares membership."
+        index: 0
+      types:
+        alias: Channel Types
+        description: "Mix and match channel types by providing a comma-separated list of any combination of `public_channel`, `private_channel`, `mpim`, `im`."
+        index: 1
+      exclude_archived:
+        alias: Exclude Archived?
+        description: "Set to `true` to exclude archived channels from the list."
```

From the user's perpective, there is a checkbox and not a bool field...
```suggestion
        description: "Check to exclude archived channels from the list."
```